app/utils/translations/poeditor/group.ts

```diff
@@ -0,0 +1,67 @@
+export {};
+import * as fs from "graceful-fs";
```

Comment:
`const fs = require("fs");` like in `generate-translation-files.js`, no need to add a devDep I believe.
If so, use `yarn add -D graceful-fs` and remove the ^ in `package.json`

Reply:
ESLint doesn't like `require`, I'll see how it does with the CI test.

Reply:
Actually, I'll use `import * as fs from "fs";`, for some reason IntelliSense didn't include `fs` when I wrote the code.